### PR TITLE
[feat] View All Items Info; Tailwind colors cleanup

### PR DESCRIPTION
**Pull request by raj-kumar-litmus**
1. Hexadecimal colour codes changed into configurable strings;
2. View All line items implemented in Order details.
3. Icon added to the alert toast shown for Auto Refresh

### CODE REVIEW

File: src/components/itemInformationsPopUp.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { FC } from "react";
+import { CommerceItemData } from "../@types/OrderDetails";
+import Card from "./common/Card";
+import CustomDialog from "./common/customdialog";
+import { ITEMS_INFO } from "../constants/appConstants";
+
+type Props = {
+  lineitems: CommerceItemData[];
+  openPromotionsPopup: boolean;
```
Comment (thasnim-cm) [inline src/components/itemInformationsPopUp.tsx]: prop names should be openItemInformationsPopup and setOpenItemInformationsPopup

File: src/components/orderDetails/OrderDetails.tsx [hunk 2]
```diff
@@ -303,17 +318,19 @@ const OrderDetails: React.FC = () => {
         </div>
       </div>
 
-        <div className="gridNoGapRounded grid-cols-1 mb-4 p-0 sm:py-4 sm:px-6 bg-[#161A1D] sm:bg-[#22262C]">
-        <div className="flex justify-between border-none !bg-[#161A1D] sm:!bg-inherit">
-          <span className="w-3/4 sm:w-full !text-lg !text-[#F2F2F2] font-bold !bg-[#161A1D] sm:!bg-inherit">
+      <div className="gridNoGapRounded grid-cols-1 mb-4 p-0 sm:py-4 sm:px-6 bg-black-100 sm:bg-black-200">
+        <div className="flex sticky top-0 justify-between border-none !bg-black-100 sm:!bg-inherit">
```
Comment (thasnim-cm) [inline src/components/orderDetails/OrderDetails.tsx]: sticky, top-0 properties are only for mobile screen right?

File: src/components/orderDetails/OrderDetails.tsx [hunk 3]
```diff
@@ -68,13 +71,19 @@ const OrderDetails: React.FC = () => {
...
     useState<boolean>(false);
   const [openOrderStatusPopup, setOpenOrderStatusPopup] =
     useState<boolean>(false);
+  const [openItemInformationPopup, setItemInformationPopup] =
+    useState<boolean>(false);
   const navigate = useNavigate();
-
+  const { width } = useScreenSize();
   const { orderId } = useParams<{ orderId: string }>();
+  const MOBILE_MIN_ORDER_LIST = 5; // todo. change to 5.
```
Comment (AdithyaShaj) [inline src/components/orderDetails/OrderDetails.tsx]: Add this in the constants file and import

**Review by thasnim-cm — APPROVED** [review]

File: tailwind.config.js [hunk 4]
```diff
@@ -5,8 +5,84 @@ export default {
...
+        yellow: {
+          100: "#FDA44F",
+        },
+      },
+      spacing: {
+        7: "7px",
+        13: "13px",
+        18: "18px",
+        44: "44px",
+        444: "444px",
```
Comment (AdithyaShaj) [inline tailwind.config.js:73]: Use rem instead of px

File: tailwind.config.js [hunk 5]
```diff
@@ -5,8 +5,84 @@ export default {
...
+        "0.5w": "0.5vw",
+        "0.5h": "0.5vh",
+      },
+      borderRadius: {
+        8: "8px",
+      },
+      fontSize: {
+        14: "14px",
+      },
```
Comment (AdithyaShaj) [inline tailwind.config.js:82]: Can we keep all the px values under one category? The classname would not make sense if 14px needs to be used in places other than font size

Comment (AdithyaShaj) [inline tailwind.config.js:90]: Can we keep all the px values under one category? The classname would not make sense if 14px needs to be used in places other than font size

Comment (raj-kumar-litmus) [reply tailwind.config.js:90]: It's not upto us to decide these things.
These categories are decided by tailwind.
We can simply use them to extend our requirements.

**Review by AdithyaShaj — APPROVED** [review]

**Pull request merged**